tests/utils/interpolation.py

```diff
@@ -157,10 +157,10 @@ def linear_interpolation_with_extrapolation(x, y, x_new):
     """Linear interpolation with extrapolation.
 
     Args:
-        x (np.ndarray): 1d array of shape (n,) containing the x-values.
-        y (np.ndarray): 1d array of shape (n,) containing the y-values
+        x : 1d array of shape (n,) containing the x-values.
+        y : 1d array of shape (n,) containing the y-values
             corresponding to the x-values.
-        x_new (np.ndarray or float): 1d array of shape (m,) or float containing
+        x_new: 1d array of shape (m,) or float containing
```

Comment:
Can you add the input types back to the docstrings? Is it ```np.ndarray```, ```jnp.ndarray```, and ```float```?

Reply:
POlished!